tools/keytools/sign.c

```diff
@@ -2072,6 +2072,44 @@ int main(int argc, char** argv)
             }
             CMD.custom_tlvs++;
             i += 2;
+        } else if (strcmp(argv[i], "--custom-tlv-string") == 0) {
+            int p = CMD.custom_tlvs;
+            uint16_t tag, len;
+            uint32_t j;
+            if (p >= MAX_CUSTOM_TLVS) {
+                fprintf(stderr, "Too many custom TLVs.\n");
+                exit(16);
+            }
+            if (argc < (i + 2)) {
+                fprintf(stderr, "Invalid custom TLV fields. \n");
+                exit(16);
+            }
+			tag = (uint16_t)arg2num(argv[i + 1], 2);
```

Comment:
Indent issue on these two lines. Perhaps tabs vs spaces?

Reply:
I believe that is fixed now. Thank you.